bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/VerisureSession.java

```diff
@@ -334,14 +350,30 @@ private ContentResponse postVerisureAPI(String url, String data, boolean isJSON)
             }
         }
         request.header("Accept", "application/json");
-        if (!data.equals("empty")) {
-            request.content(new BytesContentProvider(data.getBytes(StandardCharsets.UTF_8)),
-                    "application/x-www-form-urlencoded; charset=UTF-8");
+
+        if (url.contains(AUTH_LOGIN)) {
+            request.header("APPLICATION_ID", "OpenHAB Verisure");
+            String basicAuhentication = Base64.getEncoder().encodeToString((userName + ":" + passWord).getBytes());
+            request.header("authorization", "Basic " + basicAuhentication);
+            logger.trace("Basic Auth {}", basicAuhentication);
```

Comment:
For security reason, I would avoid such log.

Reply:
Removed it!